hack/mkdocs-copy-geps.py

```diff
@@ -15,10 +15,86 @@
 import shutil
 import logging
 from mkdocs import plugins
+import yaml
+import pandas
+from fnmatch import fnmatch
+import glob
 
 log = logging.getLogger('mkdocs')
 
 @plugins.event_priority(100)
 def on_pre_build(config, **kwargs):
     log.info("copying geps")
     shutil.copytree("geps","site-src/geps", dirs_exist_ok=True)
+
+    # calling to get the conformance reports generated
+    yamlReports = getYaml()
+    
+    generate_tables(yamlReports)
+
+
+
+# NOTE: will have to be updated if new features are added
+httproute_extended_conformance_features_list = ['HTTPRouteBackendRequestHeaderModification',"HTTPRouteQueryParamMatching",'HTTPRouteMethodMatching',"HTTPRouteResponseHeaderModification","HTTPRoutePortRedirect","HTTPRouteSchemeRedirect","HTTPRoutePathRedirect","HTTPRouteHostRewrite","HTTPRoutePathRewrite","HTTPRouteRequestMirror","HTTPRouteRequestMultipleMirrors","HTTPRouteRequestTimeout", "HTTPRouteBackendTimeout","HTTPRouteParentRefPort"]
+
+
+def generate_tables(reports):
+  # experimant to making the gateway table
+ 
+  projects = reports['organization']
+
+  http_reports = reports.loc[reports["name"]=='HTTP']
+  http_reports.set_index('organization')
+  http_reports.sort_values(['organization','version'], inplace=True)
+  http_reports.drop_duplicates(subset='organization', inplace=True, keep='last')
+  
+  table = pandas.DataFrame(columns=http_reports['organization'])
+  table.insert(loc=0, column='Features', value=httproute_extended_conformance_features_list)
```

Comment:
It might be better use of the space if we come up with list of shortened feature names.  For example, just removing the HttpRoute prefix will help.  Maybe there could be a link from the shortened feature name to its reference in the API or a similar option.  

HTTPRouteBackendRequestHeaderModification could be called Header Modification and point to https://github.com/kubernetes-sigs/gateway-api/blob/e81022784e13b041a91f73725e9f3e393fccdbfc/conformance/utils/suite/features.go#L101

Reply:
This also helps if readers aren't familiar with the terms we use for our features.

Reply:
But we can leave this for the next version, as we wouldn't want to hold the first iteration.

Reply:
+1 on this being very nice to have but also not a blocker. If you run out of time to fit this into this PR, do you mind filing a follow up issue so we don't lose track of it?